client/src/pages/OnBoarding/smallComponents/ProgressBar.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from "react";
+import { MobileStepper, makeStyles } from "@material-ui/core";
+
+const ProgressBar = (props) => {
+  const classes = useStyles();
+
+  const start = props.start || 0;
+  const end = props.end;
+  const [activeStep, setActiveStep] = useState(start);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setActiveStep(start + 1);
+    }, 100);
```

Comment:
why is this necessary?

Reply:
It has something to do with how I mounted the component (which wasn't really smart). But I refactored it so the setTimeout won't be necessary now:)